ops/lightly_run.py

```diff
@@ -45,10 +45,10 @@ def create_dataset(self) -> None:
                 self.dataset_name,
                 DatasetType.VIDEOS,
             )
-            print("Created dataset with name: ", self.dataset_name)
+            print(f"Created dataset with name: {self.dataset_name}")
         except ValueError:
             self.client.set_dataset_id_by_name(self.dataset_name)
-            print("Dataset with name ", self.dataset_name, " already existing. Adding images to datapool")
+            print(f"Dataset with name {self.dataset_name} already existing. Adding images to datapool.")
```

Comment:
One last comment 😄 It's "Dataset with name {self.dataset_name} already exists. Adding images to datapool."